tools/make/docs.mk

```diff
@@ -111,10 +111,10 @@ docs-release-gen:
 docs-check-links:
 	@$(LOG_TARGET)
 	# Check for broken links, right now we are focusing on the v1.0.0
-	# github.com does not allow access too often, there're a lot of 429 errors
+	# github.com does not allow access too often, there are a lot of 429 errors
 	# TODO: find a way to remove github.com from ignore list
 	# TODO: example.com is not a valid domain, we should remove it from ignore list
-	linkinator site/public/ -r --concurrency 25 -s "github.com example.com _print v0.6.0 v0.5.0 v0.4.0 v0.3.0 v0.2.0"
+	linkinator site/public/ -r --concurrency 25 --skip "github.com example.com github.io _print v0.6.0 v0.5.0 v0.4.0 v0.3.0 v0.2.0"
```

Comment:
intentionally part of this PR ?

Reply:
yes, the repository of open-telemetry collector: `https://open-telemetry.github.io/opentelemetry-helm-charts` is 404, but works for helm